src/components/dropdownLayouts/layout.css

```diff
@@ -0,0 +1,11 @@
+
+.category {
```

Comment:
Can you use styled-components for css. Check any of the the other examples in `*.style.js`